sdks/java/io/cdap/build.gradle

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id 'java'
+    id 'org.apache.beam.module'
+}
+
+applyJavaNature(
+        exportJavadoc: false,
+        automaticModuleName: 'org.apache.beam.sdk.io.cdap',
+)
+provideIntegrationTestingDependencies()
+enableJavaPerformanceTesting()
+
+description = "Apache Beam :: CDAP :: Java"
+ext.summary = """Apache Beam SDK provides a simple, Java-based
+interface for integration with CDAP plugins."""
+
+/** Define the list of runners which execute a precommit test.
+ * Some runners are run from separate projects, see the preCommit task below
+ * for details.
+ */
+
+allprojects {
+    repositories {
+        maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
```

Comment:
@chamikaramj We've updated the repo url due to CDAP published plugins in the sonotype repo and successfully tested the changes.

Reply:
Sonatype is fine, but why snapshots?

Reply:
CDAP published existing plugin versions snapshots in Maven format to unblock integration. 
Publishing to Maven Central repo would be feasible for them when they release new plugin versions with new features.

Reply:
As discussed offline, let's check if CDAP can publish previous released versions to the Sonatype repo and depend on that instead of depending on SNAPSHOTS.

Reply:
Hi Folks, any update on this ? Are we able to change "io.cdap.plugin:salesforce-plugins" to use 1.4.0 instead of the SNAPSHOT ?

Reply:
ETA to push plugins in Releases repo is May 6th. We should be able to update PR shortly after.

Reply:
Great! Thanks for the update.

Reply:
Seems like we don't have this dependency anymore. So this can be resolved.